src/Adapter/Presenter/Product/ProductLazyArray.php

```diff
@@ -372,7 +372,9 @@ public function getSeoAvailability()
         $seoAvailability = 'https://schema.org/';
         if ($this->product['quantity'] > 0) {
             $seoAvailability .= 'InStock';
-        } elseif ($this->product['quantity'] <= 0 && $this->product['allow_oosp']) {
+        } elseif (!Configuration::get('PS_STOCK_MANAGEMENT')) {
+            $seoAvailability .= 'InStock';
```

Comment:
So, if PS_STOCK_MANAGEMENT is disabled ; we simply tell Google (and other Search Engines), "yes, we have this item in stock"? It seems a bit like a shortcut 🤔 . What about people who work with preorders?

We should not tell to Google "this item is in stock" if we are not sure the item is in stock. Better ask @PrestaShop/product-team to be sure though.

My suggestion:
- find a better way to be sure item is in stock, when stock management is disabled
- or simply do not provide an `availability` item to Google, because we do not know?

Reply:
- if we do not provide an `availability` we will receive an error from google
- I work with disabled stock management. Whoever works like this is supposed to be because they can't have stock control or because they always have everything, right? 
 or why is someone supposed to work with disabled stock management?